### ISSUE TITLE
The abortController returns a "cancelled" error between "AbortError"

### ISSUE THREAD

**Issue by JustGitUserAccount** [axios/axios#6100]
### Describe the issue

Hi!
I am canceling a post request using AbortController, but AbortController is returning a "canceled" error between "AbortError"

How can I check that the request has been cancelled?
Do recent versions have something like "axios.isCancel(thrown)" that was in depreacated "CancelToken"?

**Comment by DigitalBrainJS** [comment]
> Do recent versions have something like "axios.isCancel(thrown)" that was in depreacated "CancelToken"?

Yes, you can still use the isCancel method in version 1.x. This is part of CanceledError functionality, not CancelToken. Additionally, each AxiosError instance has code to determine the type of error. CanceledError has a value of `'ERR_CANCELED'`.
